src/go/k8s/pkg/console/deployment.go

```diff
@@ -260,8 +253,10 @@ const (
 
 	tlsSchemaRegistryMountName = "tls-schema-registry"
 	tlsConnectMountName        = "tls-connect-%s"
+	tlsKafkaMountName          = "tls-kafka"
 
 	schemaRegistryClientCertSuffix = "schema-registry-client"
+	kafkaClientCertSuffix          = "operator-client"
```

Comment:
this should ideally not be duplicated between the cluster/console packages. not a blocker